### PR TITLE
Clarify that System & Team admins can delete posts

### PR DESCRIPTION
**Pull request by bbodenmiller**
#### Summary

When `Allow Team Administrators to edit others posts` is set to `false` one would think they also cannot delete posts. That is not the case. Clarify this in UI.

#### Screenshots

image

### CODE REVIEW

**Comment by michelengelen** [comment]
@matthewbirtch do we need your approval in this as well, since it is only a wording change?

**Comment by matthewbirtch** [comment]
I think it would be good to get @justinegeffen to review for her perspective on the wording.

At first glance, we are missing an apostrophe on the label since "others" is both plural and possessive. It should read:

`Allow Team Administrators to edit others’ posts`

Also, in the description, `other user’s posts` should be `other users’ posts`

Again, It would be helpful to get Justine's confirmation on the above.

**Comment by justinegeffen** [comment]
Thanks, @matthewbirtch, I'm reviewing now.

File: components/admin_console/admin_definition.jsx [hunk 1]
```diff
@@ -2019,7 +2019,7 @@ const AdminDefinition = {
                         label: t('admin.team.editOthersPostsTitle'),
                         label_default: 'Allow Team Administrators to edit others posts:',
                         help_text: t('admin.team.editOthersPostsDesc'),
-                        help_text_default: 'When true, Team Administrators and System Administrators can edit other user\'s posts.  When false, only System Administrators can edit other user\'s posts.',
+                        help_text_default: 'When true, Team Administrators and System Administrators can edit other user\'s posts.  When false, only System Administrators can edit other user\'s posts. Regardless of selection Team Administrators and System Administrators can delete other user\'s posts.',
```
Comment (justinegeffen) [inline components/admin_console/admin_definition.jsx:2022]:
```suggestion
                        help_text_default: 'When **true**, both Team Admins and System Admins can edit other users' posts.  When **false**, only System Admins can edit other users' posts. However, Team Admins and System Admins can always delete other users' posts.',
```

**Review by michelengelen — APPROVED** [review]
code-wise there is nothing wrong with it, so i am approving now.

File: components/admin_console/admin_definition.jsx (hunk 1 above)
Comment (bbodenmiller) [reply components/admin_console/admin_definition.jsx:2022]: @justinegeffen does markdown formatting actually work here? Doesn't ``'`` need to be escaped?

Comment (michelengelen) [reply components/admin_console/admin_definition.jsx:2022]: I am not entirely sure about markdown formatting, but the ' definitely needs to be escaped.

**Review by nevyangelova — APPROVED** [review]

**Comment by jgilliam17** [comment]
@bbodenmiller Can you please share a path to this setting in System Console? I'm not sure where to find it to verify the wording change. Thanks.

**Comment by bbodenmiller** [comment]
It's located at `/admin_console/site_config/users_and_teams` but for some reason it's not showing up in test server.

**Comment by jgilliam17** [comment]
Thank you @bbodenmiller for the path
I'll check with the team and see if I can get some help in surfacing the setting on the test server.

**Comment by jgilliam17** [comment]
Actually, this is my fault - I added an Enterprise license after failing to find the setting on my first try. 
After removing the license, setting is there 🎉 , but unfortunately the text is not updated.
@josephbaylon suggested applying changes to all translations first for updated text to show. 
@bbodenmiller Can give that a try? Thanks 🙂 
Screen Shot 2021-02-25 at 5 23 53 PM

**Comment by josephbaylon** [comment]
Just a minor correction. Only `en.json` for `admin.team.editOthersPostsDesc` is required to change for translation.

**Comment by bbodenmiller** [comment]
@jgilliam17 updated

**Comment by jgilliam17** [comment]
Thank you for a quick fix @bbodenmiller 
Text is updated, but there is still a small issue with markdown formatting.
Screen Shot 2021-02-26 at 4 42 03 PM

**Comment by bbodenmiller** [comment]
@jgilliam17 is markdown possible in settings text? I copied from suggestion given at https://github.com/mattermost/mattermost-webapp/pull/7544#discussion_r578535214. I'm fine without the markdown.

**Comment by jgilliam17** [comment]
> @jgilliam17 is markdown possible in settings text? I copied from suggestion given at [#7544 (comment)](https://github.com/mattermost/mattermost-webapp/pull/7544#discussion_r578535214). I'm fine without the markdown.

Thanks @bbodenmiller After looking around a bit, there are a few places in sys. console that have the formatted text e.g. Experimental > Features > Enable Preview Features. 
I'll follow up with the UX team on Monday and confirm if it's ok to proceed without the markdown.
Thanks again 🙂

**Comment by jgilliam17** [comment]
Apologize about delay @bbodenmiller, absolutely my fault. 
@matthewbirtch can you please advise on how to proceed? Thanks.

**Comment by bbodenmiller** [comment]
PR updated. Ready for re-review.

**Review by michelengelen — APPROVED** [review]

**Review by jgilliam17 — APPROVED** [review]
Thank you @bbodenmiller
Tested, looks good to merge.
- Verified updated help text for `Allow Team Administrators to edit others' posts` setting in Site Configurations > Users and Teams. 
@michelengelen Can you please help merge? Thanks. 🙂 
Screen Shot 2021-03-15 at 5 59 02 PM

**Comment by michelengelen** [comment]
Done @jgilliam17 🎉

**Comment by michelengelen** [comment]
Thanks for your contribution @bbodenmiller 🙇‍♂️

**Pull request merged**